### PR TITLE
update repo url to be replaced by the front-end

### PR DESCRIPTION
**Pull request by edewit**
@rhoads-zach maybe some more editing is needed

### CODE REVIEW

File: docs/topics/launcher-review-update-booster-ci.adoc [hunk 1]
```diff
@@ -21,6 +21,4 @@ $ git commit -m "Made an update"
 $ git push
 ----
 
-. Click _See it here!_ in _Creating your project on OpenShift Online_.
-
-Notice a new deployment is running! Once complete, click on the link for booster application at the top of the page and observe your changes appear.
\ No newline at end of file
+More details on interacting with your booster can be found here
```
Comment (gastaldi) [inline docs/topics/launcher-review-update-booster-ci.adoc]: The "here" should be a link?

Comment (rhoads-zach) [inline docs/topics/launcher-review-update-booster-ci.adoc]: nice catch @gastaldi, @edewit can you add the right link here?

Comment (edewit) [inline docs/topics/launcher-review-update-booster-ci.adoc]: the link to the console is also on top of the page on the "While you wait..." section maybe it's better to remove it here
https://docs.google.com/drawings/d/15rMs1Pv69OjHZLXZRtdLTGLziIX3DuEMgrhM4kHjQOw/edit

Comment (rhoads-zach) [inline docs/topics/launcher-review-update-booster-ci.adoc]: I updated it to say the following:

> More details on interacting with your booster can be found in your booster’s `README.adoc`.

What do you think of that @edewit?

Comment (edewit) [inline docs/topics/launcher-review-update-booster-ci.adoc]: sounds good :+1:

**Comment by rhoads-zach** [comment]
merged: https://github.com/openshiftio/appdev-documentation/commit/fb96d36cc2dd8a72dbdec05b8f737920b1b93df3

**Pull request closed without merging**